src/backend/shared/time.lambda-shared.ts

```diff
@@ -0,0 +1,3 @@
+export function now(): number {
+  return Date.now();
+}
```

Comment:
This could as well be:

```suggestion
export const now = Date.now;
```

Reply:
Done